src/components/EthExchanges/index.tsx

```diff
@@ -96,14 +90,15 @@ const EthExchanges = () => {
       <Text maxW="container.sm" mb={8} lineHeight={1.4} textAlign="center">
         <Translation id="page-get-eth-exchanges-intro" />
       </Text>
-      <StyledSelect
-        aria-label={t("page-get-eth-exchanges-header")}
-        className="react-select-container"
-        classNamePrefix="react-select"
-        options={exchangesByCountry}
-        onChange={handleSelectChange}
-        placeholder={placeholderString}
-      />
+      <Box w="full" maxW="container.sm">
+        <ReactSelect
+          aria-label={t("page-get-eth-exchanges-header")}
+          options={exchangesByCountry}
+          onChange={handleSelectChange}
+          placeholder={placeholderString}
+          variant="outline"
+        />
+      </Box>
```

Comment:
This is the only instance where it was originally a combobox. Should it still be a combobox?

Reply:
By combobox you are referring to a dropdown where the user can type a value and get the list filtered? if that is the case, then yes, we should support that here, as it is one of the key things to maintain.